src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,61 @@
+package duke;
+
+/**
+ * The Parser class deals with making sense of the user command.
+ */
+public class Parser {
+
+    /**
+     * Reads the user input and determines which command to execute.
+     * @param command User input.
```

Comment:
Perhaps you should leave a line between your description and your @param line?

Reply:
Alright thanks